tests/listobjects/listobjects_test.go

```diff
@@ -34,6 +35,9 @@ func testRunAll(t *testing.T, engine string) {
 	cfg := config.MustDefaultConfig()
 	cfg.Log.Level = "error"
 	cfg.Datastore.Engine = engine
+	cfg.ListObjectsDeadline = 0 // no deadline
```

Comment:
We could also just increase this to 10s, I followed the pattern in check_test.go of disabling the deadline altogether